src/nick-bot.js

```diff
@@ -0,0 +1,90 @@
+import {getEnvConfig} from "./shared.js";
```

Comment:
rename to `bot.js` to make the code / files the same on both bots (and possible future bots)?